Formula/ueberzugpp.rb

```diff
@@ -0,0 +1,33 @@
+class Ueberzugpp < Formula
+  desc "Drop in replacement for ueberzug written in C++"
+  homepage "https://github.com/jstkdng/ueberzugpp"
+  url "https://github.com/jstkdng/ueberzugpp/archive/refs/tags/v2.8.0.tar.gz"
+  sha256 "3f7e21052c3c218c436b1d2d8aedc1f02573ad83046be00b05069bfdc29bf4e2"
+  license "GPL-3.0-or-later"
+
+  depends_on "cli11" => :build
+  depends_on "cmake" => :build
+  depends_on "cppzmq" => :build
+  depends_on "nlohmann-json" => :build
+  depends_on "pkg-config" => :build
+  depends_on "fmt"
+  depends_on "libsixel"
+  depends_on "openssl@3"
+  depends_on "spdlog"
+  depends_on "tbb"
+  depends_on "vips"
+  depends_on "zeromq"
+
+  def install
+    system "cmake", "-S", ".", "-B", "build",
+      *std_cmake_args, "-Wno-dev",
+      "-DENABLE_X11=OFF", "-DENABLE_OPENCV=OFF"
+    system "cmake", "--build", "build"
+    system "cmake", "--install", "build"
+    bin.install "build/ueberzug" => "ueberzugpp"
```

Comment:
```suggestion
    bin.install "ueberzug" => "ueberzugpp"
```
I think this is what you're trying to do.

Reply:
I'm getting ENOENT errors with this change.

Reply:
Sorry, edited.

Reply:
thanks, it works now.